src/bun.js/javascript.zig

```diff
@@ -1854,7 +1854,7 @@ pub const VirtualMachine = struct {
             },
         }
 
-        if (debugger != .unspecified) {
+        if (this.debugger != null) {
```

Comment:
Can you verify this doesn't cause us to start with the debugger enabled when it's not actively in use?

Reply:
I haven't managed to get any indication this is started unless using `--inspect` or the `BUN_INSPECT` env var.

It is hard to work out however because its the `this.debugger = Debugger` stage that starts the connection (with print).